docs/coding-guidelines.md

```diff
@@ -35,7 +35,7 @@ You may observe that these conventions adhere closely to the [BEM (Blocks, Eleme
 
 The build process will split SCSS from within the blocks library directory into two separate CSS files when Webpack runs.
 
-Styles placed in a `style.scss` file will be built into `blocks/build/style.css`, to load on the front end theme as well as in the editor. If you need additional styles specific to the block's display in the editor, add them to an `editor.scss`.
```

Comment:
What would become our updated guideline for class name prefixing? I notice we're still using `block-` as prefix for classes in blocks (e.g. `.blocks-gallery-image`), despite the guideline currently recommending that anything in `editor` be prefixed as `editor-`

Reply:
I understood it like this:

The guidelines apply for the application classnames but not the frontend classnames theme authors should be aware of.